QUICK-START.md

```diff
@@ -76,7 +76,9 @@ file located in the `starting-files` directory of the extracted Ewon Flexy Cumul
 release package.
 
 Populate the `Host`, `BootstrapUsername`, `BootstrapPassword`, and `BootstrapTenant` fields with the
-corresponding values for your Cumulocity tenant.
+corresponding values for your Cumulocity tenant. The `Host` field should be populated with the 
+MQTT broker URL for your Cumulocity instance, for example, `mqtt.eu-latest.cumulocity.com` or
```

Comment:
Technically, the host is a part of a URL. Saying that the host is the URL can be confusing. For example, a url of "https://example.com:8080/pathexample" has a scheme "https://", a host "example.com" a port  "8080" and a path "examplepath".  
I would avoid saying the mqtt broker url. 
```
The `Host` field should be populated with the  MQTT broker's host. Do not include scheme or port as they are automatically selected to be https:// and 883
```

Reply:
The examples are really good. Keep those.

Reply:
Updated